pkg/grpc/api.go

```diff
@@ -620,7 +619,7 @@ func (a *api) InvokeActor(ctx context.Context, in *runtimev1pb.InvokeActorReques
 		return &runtimev1pb.InvokeActorResponse{}, err
 	}
 
-	req := invokev1.NewInvokeMethodRequest(fasthttp.MethodPut)
```

Comment:
Nice try @yaron2  :)

Reply:
@artursouza Can an e2e test be added for this or tracked?

Reply:
We will as part of the major effort to rewrite actor communication to GRPC.